scipy/signal/_upfirdn_apply.pyx

```diff
@@ -81,10 +81,6 @@ def _apply(np.ndarray data, DTYPE_t [::1] h_trans_flip, np.ndarray out,
     cdef DTYPE_t *out_ptr
     cdef int retval
 
-    # _apply_axis_inner assumes data and out are C-contiguous
-    data = np.ascontiguousarray(data)
-    out = np.ascontiguousarray(out)
```

Comment:
Isn't Cython already enforcing that due to the `np.ndarray data`, etc. specified in the function declaration?